docs/built_rst/tables/elements/feature_identifier.rst

```diff
@@ -0,0 +1,10 @@
+.. This file is auto-generated.  Do not edit it by hand!
+
++--------------+---------------+--------------+--------------+------------------------------------------+------------------------------------------+
+| Tag          | Data Type     | Required?    | Repeats?     | Description                              | Error Handling                           |
++==============+===============+==============+==============+==========================================+==========================================+
+| Index        | ``xs:string`` | **Required** | Single       | The index value for the shapefile        | If the field is invalid, then the        |
```

Comment:
WDYT about making this optional? In the future I could imagine a FeatureIdentifier element containing various different ways to refer to a shape (e.g. attribute-based identifier). In that case, it would be reasonable to provide the index or an attribute, in which case both would need to be optional in the spec to allow that flexibility.

Reply:
Sounds good to me.

Reply:
Also what if the dereferenced `ExternalFile` only represents a single polygon? Should `FeatureIdentifier` be required at all?

Reply:
Yes, FeatureIdentifier should still be required in that case. The ExternalGeospatialFeature is a reference to a single polygon within an external file containing one or more polygons. The FeatureIdentifier is how we know which single polygon to grab from that file. 

It's worth nothing that even if we make FeatureIdentifier.Index optional, one or more resolvable values is still required in a FeatureIdentifier element. This is a constraint that will need to be enforced in code to check 1) there is a usable value in each FeatureIdentifier and 2) each successfully resolves to one or more polygons in an external file.